app/views/pages/getting_started2.html.erb

```diff
@@ -58,9 +58,10 @@
 
         <h4><%= t '.activity' %></h4>
         <p><%= t('.activity_ex').html_safe %></p>
+        <p><%= t('.activity_ex2').html_safe %></p>
 
         <h4><%= t '.mentions' %></h4>
-        <p><%= t('.mentions_ex', part4_link: link_to(t('pages.tutorials.part4'), connecting_path)).html_safe %></p>
+        <p><%= t('.mentions_ex', part5_link: link_to(t('pages.tutorials.part5'), sharing_path)).html_safe %></p>
```

Comment:
We could keep `part4_link` here (for other languages) and change `part5_link` to `part_link` for english (to be safe in the future if parts are re-ordered again)? Once other languages are updated we can remove `part4_link` here.

What do others think?

Reply:
See comment on https://github.com/diaspora/diaspora-project-site/pull/107/files#diff-37c9b835927e3c363588f172de9bb6faR329 - this method has worked fine up til now. I'm not sure why there now appears to be a problem.

Reply:
> this method has worked fine up til now

The method works as long as you don't change the keys here. The problem is, the translation now (old) contain `%{part4_link}` and there is a `part4_link` parameter here.  You now changed that and only add a `part5_link` parameter here. That works for english, because the new english translation need a `%{part5_link}` to be replaced. But all other languages still contain the `%{part4_link}` and you removed the parameter, so they don't know how to replace `%{part4_link}` anymore. So you either need to replace `%{part4_link}` in all languages on WTI or you need to still provide a `part4_link` parameter here, so old translations still know how to replace it in the text.

So everytime you change something with `%{...}` in the translations and remove the parameter for that, every language still needing this parameter fails (that was always the case).

Reply:
You can change the value of the `part4_link` parameter, so it shows "Part 5" and it links to part 5, but that would be confusing in the future. That's why I suggest a generic `part_link` parameter, that doesn't need to be changed when we re-order the parts.

Reply:
Aha. Now I see the problem. Thanks for explaining. We should definitely sort this out.

I'll wait for Dennis's confirmation about using this approach, as this website is his creation, and then start making the necessary changes.

@denschub, will there be an automatic way of globally replacing `{part[n]_link}` with `{part_link}` in WTI, or would each occurrence have to be changed manually in each language? If the former, I propose that I change all such links in `en.yml` in this PR and they are globally replaced in WTI before the translations are pulled, so that the links are future-proofed now; if it would need to do manually, I propose that I revert the change to this line so that this PR can be merged and pushed without breaking translations, and then open another PR changing all these links which can be merged to `master` but not pushed to the live site, and then get on (hopefully with help from our team of translators) with changing all occurrences in all languages on WTI so that we're ready for the next push to the live site.

How does that sound?

Reply:
WTI has some batch processing functions to do things for all languages, but I don't know if you can search/replace (I can check that later). And I think you can only change one text at once, not all texts for all languages, so you would at least change this for every `{part[n]_link}` that you want to replace.

If you want to do this in two PRs, you can already start changing it in this PR. As long as you still keep the old `part4_link` parameter in this line, everything still works (you can have both, `part_link` for new translations and `part4_link` for old translations). Then you can already use `part_link` in new/changed translations from this PR.

Reply:
OK, thanks. 

> As long as you still keep the old part4_link parameter in this line, everything still works (you can have both, part_link for new translations and part4_link for old translations).

Yes, thanks, that's what I was thinking of by 'revert'. I just couldn't think of the best way to explain it. 

I think I've fixed all occurrences now.